### PR TITLE
Docs/upgrade v1.2

### PR DESCRIPTION
**Pull request by orichh**
## What kind of change does this PR introduce?

Update v1.2.0 upgrade docs and blog with clarifying notes for Pro (paid) plan projects 

## What is the current behavior?

[Docs](https://supabase.com/docs/guides/platform/migrating-and-upgrading-projects#upgrade-your-project): states upgrade instructions, but only makes note for the free tier.

[Blog Post](https://supabase.com/blog/whats-new-in-pg-graphql-v1-2): states features of v1.2.0, but does not indicate how to access these features in existing projects

## What is the new behavior?

Docs - Add to the existing note with clarification on how to upgrade existing Pro-plan projects to v1.2.0.

Blog - Add note with link to upgrade documentation

## Additional context
Addresses this issue: https://github.com/supabase/supabase/issues/13905

Also removed unnecessary trailing whitespace in the blog post (auto-formatted using prettier).

### CODE REVIEW

File: apps/www/_blog/2023-04-21-whats-new-in-pg-graphql-v1-2.mdx [hunk 1]
```diff
@@ -15,18 +15,20 @@ thumb: launch-week-6/pggraphql/og-pg-graphql.png
 ---
 
 It's been 4 months since the [1.0.0 release of pg_graphql](https://supabase.com/blog/pg-graphql-v1).
-Since then, we’ve pushed several features to improve the APIs that `pg_graphql` produces. 
+Since then, we’ve pushed several features to improve the APIs that `pg_graphql` produces.
 
 In this article, we’ll walk through those features and show examples of each.
 
+<Admonition type="note">These features are only available after upgrading your project to v1.2.0. For help with upgrading, please review the documentation [here](https://supabase.com/docs/guides/platform/migrating-and-upgrading-projects).</Admonition>
```
Comment (gregnr) [inline apps/www/_blog/2023-04-21-whats-new-in-pg-graphql-v1-2.mdx:22]: We don't currently have this component available on `www` (where the blog is hosted). Let's just stick to regular markdown for now.

Comment (richardhyesungo) [reply apps/www/_blog/2023-04-21-whats-new-in-pg-graphql-v1-2.mdx]: @gregnr 

I removed the component and replaced it with a div styling which was used here: `apps/www/_blog/2023-04-14-supabase-studio-2.0.mdx`

this is what it'll look like
Screen Shot 2023-04-23 at 12 21 26 AM

let me know if you just want to remove the div entirely and have it as plain text instead.

File: apps/www/_blog/2023-04-21-whats-new-in-pg-graphql-v1-2.mdx [hunk 2]
```diff
@@ -15,18 +15,24 @@ thumb: launch-week-6/pggraphql/og-pg-graphql.png
...
 
 It's been 4 months since the [1.0.0 release of pg_graphql](https://supabase.com/blog/pg-graphql-v1).
-Since then, we’ve pushed several features to improve the APIs that `pg_graphql` produces. 
+Since then, we’ve pushed several features to improve the APIs that `pg_graphql` produces.
 
 In this article, we’ll walk through those features and show examples of each.
 
+<div className="bg-gray-300 rounded-lg px-6 py-2 italic">
+
+📢 These features are only available after upgrading your project to v1.2.0. For help with upgrading, please review the documentation [here](https://supabase.com/docs/guides/platform/migrating-and-upgrading-projects).
```
Comment (MildTomato) [inline apps/www/_blog/2023-04-21-whats-new-in-pg-graphql-v1-2.mdx:24]: this probably need to be more clear that its referring to "Graphql" versioning. 
It also suggests "upgrading", but then links to Supabase migrating and upgrading.. I assume what it needs to really suggest is upgrading Postgres ?

**Review by MildTomato — DISMISSED** [review]

File: apps/www/_blog/2023-04-21-whats-new-in-pg-graphql-v1-2.mdx (hunk 2 above)
Comment (MildTomato) [reply apps/www/_blog/2023-04-21-whats-new-in-pg-graphql-v1-2.mdx:24]: cc @gregnr

Comment (MildTomato) [reply apps/www/_blog/2023-04-21-whats-new-in-pg-graphql-v1-2.mdx:24]: cc @olirice

Comment (gregnr) [reply apps/www/_blog/2023-04-21-whats-new-in-pg-graphql-v1-2.mdx:24]: Good catch @MildTomato.

@olirice I can update this - is this PG version correct: `15.1.0.63`?

https://github.com/supabase/postgres/commit/ccd2e902cf528a59623b2924c12053e5138681ff

Comment (olirice) [reply apps/www/_blog/2023-04-21-whats-new-in-pg-graphql-v1-2.mdx:24]: thanks, yes it is `15.1.0.63`

Comment (gregnr) [inline apps/www/_blog/2023-04-21-whats-new-in-pg-graphql-v1-2.mdx:24]:
```suggestion
📢 These features are only available on projects with Postgres version `15.1.0.63` or higher. For help with upgrading, please review the [migrating and upgrading projects guide](https://supabase.com/docs/guides/platform/migrating-and-upgrading-projects).
```

File: apps/www/_blog/2023-04-21-whats-new-in-pg-graphql-v1-2.mdx (hunk 1 above)
Comment (gregnr) [reply apps/www/_blog/2023-04-21-whats-new-in-pg-graphql-v1-2.mdx]: Looks great like this - thanks!

File: apps/www/_blog/2023-04-21-whats-new-in-pg-graphql-v1-2.mdx (hunk 2 above)
Comment (gregnr) [reply apps/www/_blog/2023-04-21-whats-new-in-pg-graphql-v1-2.mdx]: Updated 👍

**Review by gregnr — APPROVED** [review]

**Pull request merged**